docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md

```diff
@@ -83,9 +85,36 @@ Each initial configuration file of all services contains `local_config`. The def
 
     It is not recommended to modify the value of `local_config` to `false`. If modified, the Nebula Graph service will first read the cached configurations, which may cause configuration inconsistencies between clusters and cause unknown risks.
 
+### Configuration files for clusters installed with Docker Compose
+
+For clusters installed with Docker Compose, the configuration file's default installation path of the cluster is `<install_path>/nebula-docker-compose/docker-compose.yaml`. The parameters in the `command` field of the file are the start parameters for each service.  
```

Comment:
"start parameters" looks a bit strange here.

Reply:
> 

has the same meaning with Launch Parameters. will use launch instead